.gitignore

```diff
@@ -9,6 +9,7 @@ Thumbs.db
 
 # composer
 /vendor
+composer.lock
```

Comment:
When I tried to setup project on different environment - with software versions lower than on environment where composer.lock was built - have to manually delete composer.lock in order to setup project with composer.json

Initial dev env - PHP 7.0
Second dev end PHP 5.6 - have to delete composer.lock in it

Or, may be I am doing something wrong, don't know
Anyway< do not insist on this change - it's minor in my opinion